receiver/sqlqueryreceiver/metrics.go

```diff
@@ -29,6 +29,24 @@ func rowToMetric(row stringMap, cfg MetricCfg, dest pmetric.Metric, startTime pc
 	dest.SetUnit(cfg.Unit)
 	dataPointSlice := setMetricFields(cfg, dest)
 	dataPoint := dataPointSlice.AppendEmpty()
+	if cfg.StartTsColumn != "" {
+		if val, found := row[cfg.StartTsColumn]; found {
```

Comment:
Shouldn't this fail in case the column isn't found?

Reply:
Right, thanks for pointing this out Juraci. I agree this should fail when the column is not found, to make it consistent with the behavior of `attribute_columns` property, which also fails when the column is not found for an attribute.